pyaedt/modeler/circuits/PrimitivesEmit.py

```diff
@@ -712,6 +712,22 @@ def bands(self):
         band_nodes = self.get_prop_nodes({"Type": "Band"})
         return band_nodes
 
+    def band_node(self, band_name):
+        """Get the specified band node from this radio.
+
+        Parameters
+        ----------
+        band_name : name of the desired band node.
+
+        Returns
+        -------
+        band_node : Instance of the band node."""
+        band_nodes = self.bands()
+        for node in band_nodes:
+            if band_name == node.props["Name"]:
+                return node
+        return False
```

Comment:
Minor, but if the return type is an object and the expected object isn't found, `None` (no object) seems a little more fitting than False.

```suggestion
        return None
```